.github/workflows/ci.yml

```diff
@@ -31,8 +31,7 @@ jobs:
 
       - name: Install tools
         run: |
-          dotnet tool install dotnet-format --version 4.1.131201 --tool-path tools
```

Comment:
Is the dotnet-format tool included by default so we don't need to have this line anymore?

Reply:
Yes. But we are also not using it in CI.